core/src/main/java/org/mobilitydata/gtfsvalidator/validator/ValidatorLoader.java

```diff
@@ -215,7 +219,7 @@ private void addFileValidator(Class<? extends FileValidator> validatorClass) {
    * @return a new validator
    */
   @SuppressWarnings("unchecked")
-  private <T> T createValidator(Class<T> clazz, Function<Class<?>, Object> provider)
+  private <T> T createValidator(Class<T> clazz, Function<Class<?>, Object> provider, Function<Class<?>, Object> otherProvider)
```

Comment:
@aababilov @barbeau As an attempt to inject multiple dependencies (here `GtfsFeedName` and `CurrentDateTime`) I proceeded that way. Is this best practice?

Reply:
💡 `GtfsFeedName` will be changed to `CountryCode` after integrating changes from #851